homeworks/yehor.honcharenko_NOcode-developer/GamesProject/About/style.css

```diff
@@ -0,0 +1,181 @@
+body {
+  width: 100%;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.wrapper {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  min-height: 100vh;
+}
+
+/* Header */
+
+header {
+  grid-row: 1 / 2;
+  background: #EF4934;
+  padding: 8px 0;
+}
+
+.nav__container {
+  max-width: 1280px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.logoTop {
+  width: 64px;
+  height: 64px;
+}
+
+ul {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-gap: 50px;
+  justify-content: end;
+}
+
+li {
+  list-style: none;
+}
+
+.navLink {
+  text-decoration: none;
+  color: #FFF;
+  font: 500 16/24px Roboto;
+}
+
+.navLink:hover {
+  text-decoration: underline;
+}
+
+.navLink:target {
+  text-decoration: underline;
+}
+
+/* Content */
+
+main {
+  grid-row: 2 / 3;
+  max-width: 1280px;
+  margin: 0 auto;
+}
+
+.content {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-areas: "img text";
+  grid-gap: 48px;
+  max-width: 100%;
+  padding: 48px 0;
+}
+
+.img-about {
+  grid-area: img;
+  width: 100%;
+  height: auto;
+}
+
+.aboutMe {
```

Comment:
Please choose one style of writing classes in css:
- `.dash-case`(most common for static web pages)
- `.camelCase`(is using for css modules in React)
After you should use this style thru all project. Now you've mixed two of them:
```css
.img-about {}
.aboutMe {
```

Reply:
fixed / done ✅